src/status_im2/common/parallax/whitelist.cljs

```diff
@@ -0,0 +1,11 @@
+(ns status-im2.common.parallax.whitelist
+  (:require [native-module.core :as native-module]
+            [clojure.string :as string]))
+
+(def ^:const device-id (:device-id (native-module/get-device-model-info)))
+
+(def ^:const whitelist #{"iPhone11" "iPhone12" "iPhone13" "iPhone14"})
+
+(def whitelisted?
```

Comment:
I kinda think this should be a function, But thinking more about it I don't think that it'll ever change for a single device.

Reply:
yeah it won't change so I think it's fine as is. Discussed with @cammellos & @smohamedjavid in this pr ^ ^